protocols/kad/src/lib.rs

```diff
@@ -26,7 +26,7 @@
 //! [Identify](https://github.com/libp2p/specs/tree/master/identify) protocol might be seen as a core protocol. Rust-libp2p
 //! tries to stay as generic as possible, and does not make this assumption.
 //! This means that the Identify protocol must be manually hooked up to Kademlia through calls
```

Comment:
We need to update these docs that is no longer true. Doesn't have to be in this PR though :)